pulsar-functions/worker/src/test/java/org/apache/pulsar/functions/worker/SchedulerManagerTest.java

```diff
@@ -783,7 +784,7 @@ public void testAssignmentWorkerDoesNotExist() throws InterruptedException, NoSu
         functionMetaDataList.add(function2);
         doReturn(functionMetaDataList).when(functionMetaDataManager).getAllFunctionMetaData();
 
-        ThreadRuntimeFactory factory = new ThreadRuntimeFactory("dummy", null, "dummy");
+        ThreadRuntimeFactory factory = new ThreadRuntimeFactory("dummy", null, "dummy", ClearTextSecretsProvider.class.getName(), null);
```

Comment:
can we have some of these test pass something other than null for the config

Reply:
Changed the signature